python/lsst/ci/build.py

```diff
@@ -194,24 +194,24 @@ def _build_product(self, product, progress):
         os.chmod(buildscript, st.st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
 
         # Run the build script
-        with open(logfile, 'w') as logfp:
+        with open(logfile, 'w', encoding="utf-8") as logfp:
```

Comment:
you mean for consistency? Probably I hadn't even noticed that the rest of the code uses single quotes as I always use double quotes (from my perl heritage).